invenio_rdm_records/records/mappings/v7/rdmrecords/drafts/draft-v4.0.0.json

```diff
@@ -0,0 +1,508 @@
+              },
+              "license" : {
+                "type" : "text"
+              },
+              "scheme" : {
+                "type" : "keyword"
+              },
+              "uri" : {
+                "type" : "keyword"
+              }
+            }
+          },
+          "locations" : {
+            "properties": {
+              "features": {
+                "properties": {
+                  "centroid": {
+                    "type": "geo_point"
+                  },
+                  "geometry": {
+                    "type": "geo_shape"
+                  },
+                  "place": {
+                    "type": "text"
+                  },
+                  "identifiers": {
+                    "properties": {
+                      "identifier" : {
+                        "type" : "keyword"
+                      },
+                      "scheme" : {
+                        "type" : "keyword"
+                      }
+                    }
+                  },
+                  "description": {
+                    "type": "text"
+                  }
+                }
+              }
+            }
+          },
+          "publication_date" : {
+            "type" : "keyword"
+          },
+          "publication_date_range" : {
+            "type" : "date_range"
+          },
+          "references" : {
+            "properties" : {
+              "identifier" : {
+                "type" : "keyword"
+              },
+              "reference_string" : {
+                "type" : "text"
+              },
+              "scheme" : {
+                "type" : "keyword"
+              }
+            }
+          },
+          "related_identifiers" : {
+            "properties" : {
+              "identifier" : {
+                "type" : "keyword",
+                "copy_to" : [
+                  "related.identifier"
+                ]
+              },
+              "relation_type" : {
+                "type" : "keyword"
+              },
+              "resource_type" : {
+                "type": "object",
+                "properties": {
+                  "@v": {
+                    "type": "keyword"
+                  },
+                  "id": {
+                    "type": "keyword"
+                  },
+                  "title": {
+                    "type": "object",
+                    "dynamic": true
+                  }
+                }
+              },
+              "scheme" : {
+                "type" : "keyword"
+              }
+            }
+          },
+          "resource_type" : {
+            "type": "object",
+            "properties": {
+              "@v": {
+                "type": "keyword"
+              },
+              "id": {
+                "type": "keyword"
+              },
+              "title": {
+                "type": "object",
+                "dynamic": true
+              }
+            }
+          },
+          "subjects" : {
```

Comment:
Does this need "type": "object" ?

Reply:
No, `"type": "object"` is the default for anything that has "properties": https://www.elastic.co/guide/en/elasticsearch/reference/current/object.html